src/services/Redis.ts

```diff
@@ -1,63 +1,110 @@
 import { ClientOpts, createClient, RedisClient } from 'redis';
-import { promisify } from 'util';
+import { EventEmitter } from 'events';
 import Expire from '../types/Expire';
 import Service from '../types/Service';
 
+enum methodLabels {
+    get = 'get',
+    set = 'set',
+    del = 'del',
+}
+
 class Redis implements Service {
-    public commands: {
-        del: (key: string) => Promise<void>;
-        expire: (key: string, seconds: number) => Promise<void>;
-        expireAt: (key: string, seconds: number) => Promise<void>;
-        get: (key: string) => Promise<string>;
-        set: (key: string, value: string) => Promise<void>;
-    };
+    /** @type RedisClient */
+    private client: RedisClient = null;
+
+    /** @type ClientOpts */
+    private config: ClientOpts;
 
-    private client: RedisClient;
+    /** @type Boolean */
+    private ephemeral: boolean;
+
+    /** @type Boolean */
+    private listenersRegistered: boolean;
```

Comment:
Please double check if we need it.
Personally I hate type Docblocks ( pointless ) if you are already defining type on the language level ( super strict )....
I am not sure what is the benefit of that?

Reply:
Agreed 100% tbh. Have removed.